pennylane/operation.py

```diff
@@ -406,6 +407,9 @@ def _mod_and_round(x, mod_val):
     return str([id(d) if qml.math.is_abstract(d) else _mod_and_round(d, mod_val) for d in op.data])
 
 
+FlatPytree = tuple[tuple[TensorLike, ...], tuple[Wires, tuple[tuple[str, Any], ...]]]
```

Comment:
```suggestion
FlatPytree = tuple[tuple[TensorLike, ...], Hashable]
```

Happens to take that particular form as default behavior in the base class, but the actual interface just requires that it be hashable.  Maybe we should update the docstring to reflect that as well?